javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,45 @@
+var width = 5;
+var height = 7;
+var area = width * height;
+
+console.log('value of area:', area);
+console.log('typeof area:', typeof area);
+
+var bill = 2;
+var payment = 10;
+var change = payment - bill;
+
+console.log('value of change:', change);
+console.log('typeof change:', typeof change);
+
+var quizzes = 10;
+var midterm = 5;
+var final = 25;
+var grade = (quizzes + midterm + final) / 3;
+
+console.log('value of grade:', grade);
+console.log('typeof grade:', typeof grade);
+
+var firstName = 'Angela';
+var lastName = 'Nan';
+var fullName = firstName + ' ' + lastName;
+
+console.log('value of fullName:', fullName);
+console.log('typeof fullName:', typeof fullName);
+
+var pH = 10;
+var isAcidic = pH < 7;
+
+console.log('value of isAcidic:', isAcidic);
+console.log('typeof isAcidic:', typeof isAcidic);
+
+var headCount = 20;
+var isSparta = headCount === 300;
+
+console.log('value of isSparta:', typeof isSparta);
+console.log('typeof isSparta:', typeof isSparta);
+
+var motto = fullName;
+
+console.log('value of motto:', fullName + ' is the GOAT');
```

Comment:
this will work for the log, but you want to make sure to update the variable as well. Take a look at the += operator https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Addition_assignment for this part